portal-kernel/src/com/liferay/portal/kernel/portlet/BasePortletLayoutFinder.java

```diff
@@ -239,9 +239,15 @@ private ObjectValuePair<Long, String> _getPlidPortletIdObjectValuePair(
 			long groupId, long scopeGroupId, String portletId)
 		throws PortalException {
 
+		String[] portletTypes = {
+			LayoutConstants.TYPE_CONTENT,
+			LayoutConstants.TYPE_FULL_PAGE_APPLICATION,
+			LayoutConstants.TYPE_PORTLET
+		};
+
 		for (boolean privateLayout : Arrays.asList(false, true)) {
 			List<Layout> layouts = LayoutLocalServiceUtil.getLayouts(
-				groupId, privateLayout, LayoutConstants.TYPE_PORTLET);
+				groupId, privateLayout, portletTypes);
```

Comment:
Hi @SamZiemer 

this kind of call is normally inlined as :
```

new String[] {
			LayoutConstants.TYPE_CONTENT,
			LayoutConstants.TYPE_FULL_PAGE_APPLICATION,
			LayoutConstants.TYPE_PORTLET
	}
```

Reply:
Hi @AliciaGarciaGarcia , I've pushed another commit to inline this, thank you.